.storybook/themes/commerce.scss

```diff
@@ -396,6 +396,54 @@
 						),
 					),
 				),
+				story-carousel: (
+					components: (
+						carousel: (
+							--slides: 1,
+						),
+						carousel-actions: (
+							margin-top: 0,
+							// Need to have the setting here so desktop gets included
+						),
+						carousel-track: (
```

Comment:
Once feedback is addressed in the feature pack PR we can bring those changes over to here

Reply:
@matthewroach Thank you for this, I have integrating the change in both the themepack and in the block repo.  Confirmed in both Fusion and Storybook.